lib/util/date.tsx

```diff
@@ -6,10 +6,17 @@ export type Month = typeof monthsList[number]
 export const weekDayList = ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday'] as const
 export type WeekDay = typeof weekDayList[number]
 
-export const formatDate = (date: Date) => {
+export const formatDateTime = (date: Date) => {
```

Comment:
You can call `formatDate()` here as well and apply the time string logic afterwards